settings.gradle

```diff
@@ -71,11 +86,11 @@ if (localProperties != null) {
     String appServicesLocalPath = localProperties.getProperty(settingAppServicesPath);
 
     if (appServicesLocalPath != null) {
-        logger.lifecycle("Local configuration: substituting application-services modules from path: $appServicesLocalPath")
-
-        includeBuild(appServicesLocalPath)
+        logger.lifecycle("Enabling automatic publication of application-services from: $appServicesLocalPath")
+        def publishAppServicesCmd = ["./gradlew", "autoPublishForLocalDevelopment"]
+        runCmd(publishAppServicesCmd, appServicesLocalPath, "Published application-services for local development.")
```

Comment:
This produces no visible output while publishing the appservices components, which could leave the user wondering what's happening for many minutes if the rust code needs to be rebuilt. I think it would be better for this to send output to the same terminal as the calling app, but I need to investigate the right options to pass to make it so.